### ISSUE TITLE
TypeError Due to Duplicate 'auth' Argument in aiohttp Request when provide header  to APIChain

### ISSUE THREAD

**Issue by bugramatik** [langchain-ai/langchain#8842]
### System Info

Langchain version: 0.0.253
Python:3.11

### Who can help?

@agola11 @hwchase17 @eyurtsev

### Information

- [ ] The official example notebooks/scripts
- [ ] My own modified scripts

### Related Components

- [ ] LLMs/Chat Models
- [ ] Embedding Models
- [ ] Prompts / Prompt Templates / Prompt Selectors
- [ ] Output Parsers
- [ ] Document Loaders
- [ ] Vector Stores / Retrievers
- [ ] Memory
- [ ] Agents / Agent Executors
- [ ] Tools / Toolkits
- [X] Chains
- [ ] Callbacks/Tracing
- [X] Async

### Reproduction

1. Environment Setup:

Ensure you're using Python 3.11.
Install the necessary libraries and dependencies:

```bash
pip install fastapi uvicorn aiohttp langchai
```

2. APIChain Initialization:

Set up the APIChain utility using the provided API documentation and the chosen language model:

```python
from langchain import APIChain

chain = APIChain.from_llm_and_api_docs(api_docs=openapi.MY_API_DOCS, llm=choosen_llm, verbose=True, headers=headers)
```
3. Run the FastAPI application:

Use a tool like Uvicorn to start your FastAPI app:
```lua
uvicorn your_app_name:app --reload
```
4. Trigger the API Endpoint:

Make a request to the FastAPI endpoint that uses the APIChain utility. This could be through tools like curl, Postman, or directly from a browser, depending on how your API is set up.
Execute the Callback:

Inside the relevant endpoint, ensure you have the following snippet:
```python

with get_openai_callback() as cb:
    response = await chain.arun(user_query)
```
5. Observe the Error:

You should encounter a TypeError indicating a conflict with the auth argument in the aiohttp.client.ClientSession.request() method. Because of providing header to APIChain and running it with ```arun``` method.

### Expected behavior

Request Execution:

The chain.arun(user_query) method should interact with the intended external service or API without any issues.
The auth parameter, when used in the underlying request to the external service (in aiohttp), should be correctly applied without conflicts or multiple definitions.

**Comment by bugramatik** [comment]
If langchain/utilities/requests.py line 60 is changed like this:

```python
if self.auth:
    async with session.request(
            method, url, headers=self.headers, auth=self.auth, **kwargs
    ) as response:
        yield response
else:
    async with session.request(
            method, url, headers=self.headers, **kwargs
    ) as response:
        yield response
```

problem can be solved.

**Comment by nfcampos** [comment]
Thanks for reporting, this is fixed in #11010
